configure.ac

```diff
@@ -74,8 +72,14 @@ AC_CHECK_LIB(dl, dlopen, [],
 dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  libfabric requires libpthread.]))
-AC_CHECK_LIB(rt, clock_gettime, [],
-    AC_MSG_ERROR([clock_gettime() not found.  libfabric requires librt.]))
+
+AC_SEARCH_LIBS([clock_gettime],[rt posix4],
+		[AC_DEFINE([HAVE_CLOCK_GETTIME], [1],
+			[Define to 1 if have clock_gettime.])],
+		[AS_IF([test -z $(expr "$host_os" : '.*\(darwin\)')],
+			[AC_MSG_ERROR([clock_gettime not found.])],
```

Comment:
I only added posix4 because while I was looking for a solution to clock_gettime I found that it can be found in three places across systems: librt, libc, and libposix4. That isn't a necessary modification and I will remove it. 

Ok, instead of testing for "darwin" in host_os I'll replace it with a check for host_get_clock_service and relevant functions.